azure-pipelines.yml

```diff
@@ -0,0 +1,18 @@
+pr:
+- '*'
+
+pool:
+  vmImage: 'windows-latest'
+
+steps:
+- task: BatchScript@1
+  displayName: 'Run VsDevCmd.bat'
+  inputs:
+    filename: 'C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\Common7\Tools\VsDevCmd.bat'
```

Comment:
>C:\Program Files (x86) [](start = 15, length = 22)

Can we use %ProgramFiles(x86)% here instead of a hard-coded path? Or do we have some sort of guarantee around drive letter/install path for the build agents?

Reply:
When I changed to this, the following error showed:
"D:\a\1\s\%ProgramFiles(x86)%\Microsoft Visual Studio\2019\Enterprise\Common7\Tools\VsDevCmd.bat"
The filename, directory name, or volume label syntax is incorrect

When I use the original path:
it interprets correct and loads vsdevcmd

---
In reply to: [517491217](https://github.com/microsoft/sarif-visualstudio-extension/pull/252#discussion_r517491217) [](ancestors = 517491217)